lfucg-exactions/server/accounts/serializers.py

```diff
@@ -88,10 +88,18 @@ class AgreementSerializer(serializers.ModelSerializer):
     account_id = AccountField()
 
     agreement_type_display = serializers.SerializerMethodField(read_only=True)
+    agreement_balance = serializers.SerializerMethodField(read_only=True)
 
     def get_agreement_type_display(self, obj):
         return obj.get_agreement_type_display()
 
+    def get_agreement_balance(self, obj):
+        calculated_balance = calculate_agreement_balance(obj.id)
+
+        return {
+            'total': '${:,.2f}'.format(calculated_balance),
```

Comment:
can be simplified to just format(calculate_agreement_balance(obj.id)) since you're not using calculated_balance anywhere else